### DISCUSSION TITLE
How do we allow tab navigation for select elements outside the modal content? (navbar logo outside modal))

### DISCUSSION THREAD

**Discussion by yuchant** [reactjs/react-modal#1010] [Q&A] [answered]
I have a full screen modal with a navbar visible above it.

How can I allow tabbing there? 

```
<navbar> // this is not hidden, but can't be tabbed to.
<main> // this is hidden by modal, not visible nor tabbable as expected
<modal>
    // this is tabbable as expected
</modal>
```

I have been trying to see how the keyboard trapping works, but no luck so far.

Thank you!

**Comment by diasbruno** [comment, marked as answer]
Actually this behavior is not suitable for modal. 

When a modal is open, the focus must remain on it until the required action is done. 
This is an accessibility requirement.

It's not possible to disable this behavior on the current version.

If you have this requirement on your application, I'm afraid react-modal won't help. :(
